src/main/java/net/sf/jabref/bibtex/CamelCaser.java

```diff
@@ -1,101 +0,0 @@
-package net.sf.jabref.bibtex;
-
-import java.util.HashMap;
-import java.util.Map;
-
-class CamelCaser {
```

Comment:
That class might have been useful as a future formatter, but nevermind. We can rewrite it if needed.